CODING.md

```diff
@@ -1,3 +1,61 @@
-# iTowns coding guide
+# Development
 
-This guide gives advices and conventions on how to keep your code clean and coherent with the iTowns codebase.
+iTowns needs `nodejs` and `npm` (at least 6.x) installed on the system. See the
+[node website](https://nodejs.org) to install node and npm.
+
+## Building
+
+* Download dependencies: `npm install`
+* Run the development server:
+   * `npm start` (you can change the port: `npm start -- --port 3000`. If you get an error message that is not related to the port being in use, please update to last `lts` or stable version)
+   * Open `http://localhost:8080/`
+   * Make changes in the code, the browser will automatically reload on save
+* Build iTowns to produce a single independent script:
+   * `npm run build`
+   * The script and its source-maps are generated into the `dist/` folder
+   * You can run the built script to verify it works OK:
+      * `python -m SimpleHTTPServer 8080` (change the port at your convenience)
+      * Open `http://localhost:8080/`
+* Transpile itowns to ES5 to require it in your code:
+   * `npm run transpile`
+   * The transpiled source is in `lib/`
+   * You can then require `src/Main.js` in your code
+* Test and lint changes: see test, lint and test-examples npm script
+
+## Testing
+
+In order to runt the tests,
```

Comment:
runt? :wink: 

As said earlier, might be better to only move things around here, and update content in a separate commit (and/or PR), as it would make the changes "visible" by diffing.

Reply:
As you like ! I'm reverting back to the first states then